docs/README.md

```diff
@@ -1,14 +1,17 @@
-Here lies our documentation. After making edits, run:
+# Tendermint
 
-```
-pip install -r requirements.txt
-make html
-```
+Welcome to the Tendermint Core documentation! The introduction below provides an overview to help you navigate to your area of interest.
 
-to build the docs locally then open the file `_build/html/index.html` in your browser.
+## Introduction
 
-**WARNING:** This documentation is intended to be viewed at:
+Tendermint Core is Byzantine Fault Tolerant (BFT) middleware that takes a state transition machine - written in any programming language - and securely replicates it on many machines. In other words, a blockchain.
 
-https://tendermint.readthedocs.io
+Tendermint requires an application running over the Application Blockchain Interface (ABCI) - and comes packaged with an example application to do so. Follow the [installation instructions](./install) to get up and running quickly. For more details on [using tendermint](./using-tendermint) see that and the following sections.
 
-and may contain broken internal links when viewed from Github.
+## Networks
+
+Testnets can be setup manually on one or more machines, or automatically on one or more machine, using a variety of methods described in the [deploy testnets section](./deploy-testnets). For more information (and to join) about the Cosmos Network testnets, see [here](link).
```

Comment:
`see [here](link)` link is missing

Reply:
yeah it's a placeholder. ping @jolesbi or @nylira

Reply:
`/getting-started/full-node.md`